TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/activities/SearchFriendActivity.java

```diff
@@ -0,0 +1,94 @@
+package com.fbu.thefoodienetwork.activities;
+
+import android.os.Bundle;
+import android.util.Log;
+import android.view.KeyEvent;
+import android.view.View;
+import android.view.inputmethod.EditorInfo;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+
+import com.fbu.thefoodienetwork.adapters.FriendAdapter;
+import com.fbu.thefoodienetwork.adapters.LocationAdapter;
+import com.fbu.thefoodienetwork.databinding.ActivitySearchFriendBinding;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SearchFriendActivity extends AppCompatActivity {
+    private final static String TAG = "SearchFriend";
+    private final static String modifier = "i";
+    private final static String USERNAME_KEY = "username";
+    private final static String SCREEN_NAME_KEY = "screenName";
+    private final ParseUser CURRENT_USER = ParseUser.getCurrentUser();
+    private List<ParseUser> userList;
+    private FriendAdapter friendAdapter;
+    private ActivitySearchFriendBinding binding;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        binding = ActivitySearchFriendBinding.inflate(getLayoutInflater());
+        View view = binding.getRoot();
+        setContentView(view);
+        searchListener();
+        userList = new ArrayList<>();
+        binding.resultsRecyclerView.setLayoutManager(new LinearLayoutManager(this));
+        friendAdapter = new FriendAdapter(SearchFriendActivity.this, userList);
+        binding.resultsRecyclerView.setAdapter(friendAdapter);
+    }
+
+    private void searchListener() {
```

Comment:
Could benefit from a verb in the name similar to the above suggestions for setting listeners